koku/masu/processor/worker_cache.py

```diff
@@ -73,7 +73,7 @@ def add_task_to_cache(self, task_key):
         task_list.append(task_key)
         self.set_host_specific_task_list(task_list)
         msg = f"Added {task_key} to cache."
-        LOG.info(msg)
+        LOG.debug(msg)
```

Comment:
I think we want to always log this. We can make the statement more informative, but we need to see the cache is working as expected.